packages/appstore/src/constants/tour-steps-config-new.tsx

```diff
@@ -77,20 +77,20 @@ export const getTourStepConfigHighRisk = (): Step[] => [
         disableBeacon: true,
         disableOverlayClose: true,
     },
-    {
-        title: (
-            <React.Fragment>
-                <Text as='p' weight='bold' color='brand-red-coral'>
-                    {localize("Trader's Hub tour")}
-                    <div className='toggle-account-type__divider' />
-                </Text>
-            </React.Fragment>
-        ),
-        content: <Text as='p'>{localize('Click here if you ever need to repeat this tour.')}</Text>,
-        target: '.trading-hub-header__tradinghub--onboarding--logo',
-        disableBeacon: true,
-        disableOverlayClose: true,
-    },
+    // {
+    //     title: (
+    //         <React.Fragment>
+    //             <Text as='p' weight='bold' color='brand-red-coral'>
+    //                 {localize("Trader's Hub tour")}
+    //                 <div className='toggle-account-type__divider' />
+    //             </Text>
+    //         </React.Fragment>
+    //     ),
+    //     content: <Text as='p'>{localize('Click here if you ever need to repeat this tour.')}</Text>,
+    //     target: '.trading-hub-header__tradinghub--onboarding--logo',
+    //     disableBeacon: true,
+    //     disableOverlayClose: true,
+    // },
```

Comment:
temporary comment this out for future guidance

Reply:
Should we then leave here //TODO? or some explanation? :)

Reply:
Will leave an explaination on it 🙏